pkg/backend/httpstate/client/client_test.go

```diff
@@ -272,3 +273,89 @@ func TestGetCapabilities(t *testing.T) {
 			string(resp.Capabilities[0].Configuration))
 	})
 }
+
+func TestDeploymentSettingsApi(t *testing.T) {
```

Comment:
Since half of the fix was for yaml, can we have a test for that as well?

Reply:
fair, I added a test for the deployment file parsing (`workspace.LoadProjectStackDeployment`)